Source/Extensions/Blazorise.Splitter/JsonConverters/CamelCaseEnumJsonConverter.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Blazorise.Splitter;
+
+/// <summary>
+/// JsonConverter for enums that converts to and from a Camel Case result
+/// </summary>
+public class CamelCaseEnumJsonConverter : JsonStringEnumConverter
+{
+    /// <inheritdoc />
+    public CamelCaseEnumJsonConverter() : base( JsonNamingPolicy.CamelCase )
+    {
+    }
+}
```

Comment:
Users could have their own rules for serializing enums. Meaning it would mess with our enum names. This way, we are strictly defining how it should be serialized. We definitely need this in other places in Blazorise.

Reply:
For a moment I forgot you can't define serializer options for specific JSInterop calls. Or at least as far as I know. yea, It's fine then.